ui/Rhino/AGS/dev/AGS_form_assign_forces_cmd.py

```diff
@@ -39,13 +40,17 @@ def RunCommand(is_interactive):
                 form.diagram.edges_attribute('is_ind', False)
                 form.diagram.edges_attribute('is_ind', True, keys=edges)
                 scene.update()
+            else:
```

Comment:
always try to use "early returns"
```python
if not edges:
    break
...
```

Reply:
Here the break is in the while loop instead of end the execution of the function...